src/main/java/dinkplugin/message/Embed.java

```diff
@@ -18,6 +18,7 @@
 @Value
 @Builder
 public class Embed {
+    public static final int MAX_IMAGE_SIZE = 8_000_000; // 8 MB
```

Comment:
```suggestion
    // The max size of the image before we rescale it to fit Discords file upload limits https://discord.com/developers/docs/reference#uploading-files
    public static final int MAX_IMAGE_SIZE = 8_000_000; // 8MB
```

Technically I think the upload limit was very recently upped to 25MB, however I think this is still a reasonable change to make, as we don't want to hog the network to upload a 24MB 8K PNG imo. A separate PR can make either this number configurable (as it depends on the boost level of the server) or some setting to just disable scaling as a whole.